samcli/local/apigw/local_apigw_service.py

```diff
@@ -155,6 +156,8 @@ def create(self):
         """
         Creates a Flask Application that can be started.
         """
+        # Setting sam local start-api to respond using HTTP/1.1 instead of the default HTTP/1.0
+        WSGIRequestHandler.protocol_version = "HTTP/1.1"
```

Comment:
will this change affect the customers who use  HTTP/1.0 clients ?

Reply:
I spent some time reading about it and also tried curling HTTP/1.0 requests. I don't think this would affect the customers who use 1.0.

Reply:
is there a way to add an integration test cases for it?

Reply:
I checked the requests docs and it only allows sending HTTP/1.1 requests. This [SO](https://stackoverflow.com/questions/31728124/how-to-define-the-http-protocol-version-in-requests) seems to have a workaround to sending HTTP/1.0 requests. I can try to create integration tests using it.